openmdao/core/system.py

```diff
@@ -27,8 +28,15 @@ class _SysData(object):
     """
     def __init__(self, pathname):
         self.pathname = pathname
-        self._to_prom_name = {}
-        self._to_top_prom_name = {}
+
+        # map absolute name to local promoted name
+        self.to_prom = {}
```

Comment:
seems like it would be more consistent (and more understandable) if this were `to_prom_names`